core/src/testUtils/groovy/nu/yona/server/test/CommonAssertions.groovy

```diff
@@ -0,0 +1,272 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.test
+
+import java.time.Duration
+import java.time.LocalDate
+import java.time.ZonedDateTime
+
+import groovy.json.*
+import nu.yona.server.YonaServer
+
+class CommonAssertions extends Service
```

Comment:
Shouldn't this class be marked as `static`?

Reply:
Marking it ``static`` doesn't make a difference. If it were Java, I would add a private constructor with a comment, but that feels as overkill for a scripting language. What do you think?

Reply:
I agree, I now read that Java has no such concept.